README.md

```diff
@@ -1,229 +1,19 @@
-
-Additional steps to load `OpenMPI` on Fedora:
-
-```
-source /etc/profile.d/modules.sh
-export MODULEPATH=$MODULEPATH:/usr/share/modulefiles
-module load mpi/openmpi-x86_64
-```
-
-Compile with OpenMPI (with halo exchange):
-
-```
-mkdir build && cd build
-export CXX_COMPILER=mpicxx
-cmake -DCMAKE_BUILD_TYPE=Release -DKAHIP_ROOT=~/workspace/KaHIP/ -DHALO_EXCHANGE=On ..
-make -jN
-```
-
-To enable halo exchange set `-DHALO_EXCHANGE=On` in `CMake`. Halo exchange is a better MPI communication protocol, however, use this only for larger number of MPI tasks (> 4).
-
-### Compile with Ninja build system [Alternative to Make]
-
-0. Run `mkdir build && cd build && cmake -GNinja -DCMAKE_CXX_COMPILER=g++ ..`.
-
-1. Run `ninja`
-
-### Compile with Partio viz support
-
-Please include `-DPARTIO_ROOT=/path/to/partio/` in the cmake command. A typical cmake command would look like `cmake -DCMAKE_BUILD_TYPE=Release -DPARTIO_ROOT=~/workspace/partio/ ..`
-
-## Run tests
-
-0. Run `./mpmtest -s` (for a verbose output) or `ctest -VV`.
-
-## Run MPM
-> See [CB-Geo MPM Documentation](https://mpm.cb-geo.com/) for more detailed instructions.
-
-The CB-Geo MPM code uses a `JSON` file for input configuration. To run the mpm code:
-
-```
-   ./mpm  [-p <parallel>] [-i <input_file>] -f <working_dir> [--]
-          [--version] [-h]
-```
-
-For example:
-
-```
-export OMP_SCHEDULE="static,4"
-./mpm -f /path/to/input-dir/ -i mpm-usf-3d.json
-```
-
-Where:
-
-```
-   -p <parallel>,  --parallel <parallel>
-     Number of parallel threads
-
-   -i <input_file>,  --input_file <input_file>
-     Input JSON file [mpm.json]
-
-   -f <working_dir>,  --working_dir <working_dir>
-     (required)  Current working folder
-
-   --,  --ignore_rest
-     Ignores the rest of the labeled arguments following this flag.
-
-   --version
-     Displays version information and exits.
-
-   -h,  --help
-     Displays usage information and exits.
-```
-
-### Running the code with MPI
-
-To run the CB-Geo mpm code on a cluster with MPI:
-
-```
-mpirun -N <#-MPI-tasks> ./mpm -f /path/to/input-dir/ -i mpm.json
-```
-
-For example to run the code on 4 compute nodes (MPI tasks):
-
-```
-mpirun -N 4 ./mpm -f ~/benchmarks/3d/uniaxial-stress -i mpm.json
-```
+Please refer to [our wiki](https://github.com/geomechanics/mpm/wiki) for information on compiling, and running the code. 
 
 ## Authors
 
-Please refer to the [list of contributors to the CB-Geo MPM code](AUTHORS.md).
-
-## Citation
-
-If you publish results using our code, please acknowledge our work by quoting the following paper:
-
-Kumar, K., Salmond, J., Kularathna, S., Wilkes, C., Tjung, E., Biscontin, G., & Soga, K. (2019). Scalable and modular material point method for large scale simulations. 2nd International Conference on the Material Point Method. Cambridge, UK. [https://arxiv.org/abs/1909.13380](https://arxiv.org/abs/1909.13380)
+Please refer to the [list of contributors to the Geomechanics MPM code](AUTHORS.md).
```

Comment:
Maybe make the AUTHORS.md only us and then refer to the CB-Geo authors so that if they update it there, we don't have to update. See the one in the wiki.

Reply:
I agree. Lets 
* updated the AUTHORS.md
* removed the authors wiki page and link directly to the AUTHORS.md (like the license.md)

Reply:
Sounds good!